sdk/CMakeLists.txt

```diff
@@ -84,23 +84,15 @@ option(EXECUTORCH_SEPARATE_FLATCC_HOST_PROJECT
 )
 
 if(EXECUTORCH_SEPARATE_FLATCC_HOST_PROJECT)
-  # Add the host project. We build this separately so that we can generate
-  # headers on the host during the build, even if we're cross-compiling the
-  # flatcc runtime to a different architecture.
```

Comment:
Please preserve this comment to help readers understand why this is built in a special way.

Reply:
Will do, thanks.